inc/scroom/layeroperations.hh

```diff
@@ -35,7 +36,7 @@ private:
   ColormapProvider::Ptr colormapProvider;
 
 public:
-  static Ptr create(ColormapProvider::Ptr colormapProvider);
+  static LayerOperations::Ptr create(ColormapProvider::Ptr colormapProvider);
```

Comment:
Ambiguity is probably with `PipetteLayerOperations::Ptr`, which is currently unused. So you might as well delete that `typedef`.

Reply:
I am not sure whether this can be removed. Removing this would also change the way in which the `dynamic_pointer_cast` is done at the top of the implementation of `getPixelAverages()`. If we remove this cast (which i am not really sure if we can), we can probably remove that `typedef`